core/src/br/cefetmg/games/screens/PlayingGamesScreen.java

```diff
@@ -138,11 +139,12 @@ private void advance() {
         if (this.state != PlayScreenState.PLAYING) {
             return;
         }
-
+        
         if (this.sequencer.hasNextGame()) {
             this.currentGame = this.sequencer.nextGame();
             hud.setGameIndex(sequencer.getGameNumber());
             Gdx.input.setCursorCatched(currentGame.shouldHideMousePointer());
+            Gdx.input.setCursorPosition(Config.WORLD_WIDTH/3, Config.WORLD_HEIGHT/3);
```

Comment:
`WORLD_WIDTH`/`HEIGHT` são as dimensões virtuais do mundo, mas o tamanho da janela pode ser diferente.
Veja aqui: http://stackoverflow.com/questions/19940295/how-to-get-screen-resolution-of-an-android-device-using-libgdx